src/battle_util.c

```diff
@@ -4610,7 +4610,7 @@ u32 AbilityBattleEffects(u32 caseID, u32 battler, u32 ability, u32 special, u32
             break;
         case ABILITY_TERAFORM_ZERO:
             if (!gSpecialStatuses[battler].switchInAbilityDone
-                && gBattleMons[battler].species == SPECIES_TERAPAGOS_STELLAR)
+                 && gBattleMons[battler].species == SPECIES_TERAPAGOS_STELLAR)
```

Comment:
This align looks out of place.

Reply:
It's consistent with the other ones

Reply:
It's not consisten with 4818-4819, for example

Reply:
It's not consistent with 4818-4819, for example

Reply:
Oh, a lot of aligns around here are inconsistent with eachother